crates/daphne/src/messages/request.rs

```diff
@@ -24,15 +25,79 @@ macro_rules! impl_req_body {
     };
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(any(test, feature = "test-utils"), derive(deepsize::DeepSizeOf))]
+pub struct AggregationJobRequestHash(Vec<u8>);
+
+impl AggregationJobRequestHash {
+    pub fn get(&self) -> &[u8] {
+        &self.0
+    }
+
+    fn hash(bytes: &[u8]) -> Self {
+        Self(
+            ring::digest::digest(&ring::digest::SHA256, bytes)
+                .as_ref()
+                .to_vec(),
+        )
+    }
+}
+
```

Comment:
```suggestion

/// An [`AggregationJobInitReq`] and its hash. Used by the helper to prevent the parameters of an aggregation job from changing.
```